package-lock.json

```diff
@@ -371,45 +382,35 @@
         "@types/unist": "*"
       }
     },
+    "@typescript-eslint/experimental-utils": {
+      "version": "1.9.0",
+      "resolved": "https://registry.npmjs.org/@typescript-eslint/experimental-utils/-/experimental-utils-1.9.0.tgz",
+      "integrity": "sha512-1s2dY9XxBwtS9IlSnRIlzqILPyeMly5tz1bfAmQ84Ul687xBBve5YsH5A5EKeIcGurYYqY2w6RkHETXIwnwV0A==",
+      "dev": true,
+      "requires": {
+        "@typescript-eslint/typescript-estree": "1.9.0"
+      }
+    },
     "@typescript-eslint/parser": {
-      "version": "1.7.0",
-      "resolved": "https://registry.npmjs.org/@typescript-eslint/parser/-/parser-1.7.0.tgz",
-      "integrity": "sha512-1QFKxs2V940372srm12ovSE683afqc1jB6zF/f8iKhgLz1yoSjYeGHipasao33VXKI+0a/ob9okeogGdKGvvlg==",
+      "version": "1.9.0",
+      "resolved": "https://registry.npmjs.org/@typescript-eslint/parser/-/parser-1.9.0.tgz",
+      "integrity": "sha512-CWgC1XrQ34H/+LwAU7vY5xteZDkNqeAkeidEpJnJgkKu0yqQ3ZhQ7S+dI6MX4vmmM1TKRbOrKuXc6W0fIHhdbA==",
       "dev": true,
       "requires": {
-        "@typescript-eslint/typescript-estree": "1.7.0",
+        "@typescript-eslint/experimental-utils": "1.9.0",
+        "@typescript-eslint/typescript-estree": "1.9.0",
         "eslint-scope": "^4.0.0",
         "eslint-visitor-keys": "^1.0.0"
-      },
-      "dependencies": {
-        "eslint-scope": {
```

Comment:
Hi @NishealJ, I have a couple of questions here:
1. There seem to be deletions in this file. Is this the case or have some dependencies been moved? If the latter, please revert them to their original positions.
2. There are multiple versions of eslint-scope listed in this file as dependencies (4.0.3 and 3.7.1). Why? Are there any other such dependencies? Can you please check this?

Reply:
Hi @apb7,
yes, I'll provide the testing docs for this.
package-lock.json was auto-generated after the upgrade, I believe there are deletions in package-lock.json because, after the upgrade, libraries might have changed their requirements.
Also,
eslint-scope is mentioned twice because package-lock contains all the libraries and their requirements, if 2 or more libraries require eslint-scope it will be mentioned more than 2 time.

I'll completely test this and provide a testing doc thanks!

Reply:
>yes, I'll provide the testing docs for this.

Hmm. I was looking for the manual tests you must have carried out locally when upgrading this.

>package-lock.json was auto-generated after the upgrade, I believe there are deletions in package-lock.json because, after the upgrade, libraries might have changed their requirements.

Okay, thanks for the explanation.

>eslint-scope is mentioned twice because package-lock contains all the libraries and their requirements, if 2 or more libraries require eslint-scope it will be mentioned more than 2 time.

I agree -- if there are two or more libraries that require eslint-scope, it will be mentioned more than two times. I was more concerned regarding two different _versions_ of eslint-scope...

>I'll completely test this and provide a testing doc thanks!

Okay, if this PR is untested, can you please do the following:
1. Label this as a WIP and add to the PR explanation whatever is left to do.
2. Assign the PR to yourself until it remains a WIP. As soon as you are done with the PR, please re-assign it to me so that I can review it.

Thanks!